nav2_bringup/launch/tb3_simulation_launch.py

```diff
@@ -134,11 +135,7 @@ def generate_launch_description():
 
     declare_world_cmd = DeclareLaunchArgument(
         'world',
-        # TODO(orduno) Switch back once ROS argument passing has been fixed upstream
-        #              https://github.com/ROBOTIS-GIT/turtlebot3_simulations/issues/91
-        # default_value=os.path.join(get_package_share_directory('turtlebot3_gazebo'),
-        # worlds/turtlebot3_worlds/waffle.model')
-        default_value=os.path.join(bringup_dir, 'worlds', 'world_only.model'),
+        default_value=os.path.join(aws_dir, 'worlds', 'small_warehouse', 'small_warehouse.world'),
```

Comment:
oh - should we be using the no roof model instead? That way people can actually see what's happening!

Reply:
across all

Reply:
Done.